packages/pwa-kit-dev/package.json

```diff
@@ -39,6 +39,7 @@
     "@babel/cli": "^7.21.0",
     "@babel/core": "^7.21.3",
     "@babel/eslint-parser": "^7.21.3",
+    "@babel/node": "^7.22.5",
```

Comment:
we already have a 3pp filed for this I think?

Reply:
Multiple @babel/node 3PP approvals found in [GUS](https://gus.lightning.force.com/one/one.app#eyJjb21wb25lbnREZWYiOiJmb3JjZVNlYXJjaDpzZWFyY2hQYWdlRGVza3RvcCIsImF0dHJpYnV0ZXMiOnsidGVybSI6IkBiYWJlbC9ub2RlIiwic2NvcGVNYXAiOnsidHlwZSI6IlRPUF9SRVNVTFRTIiwibmFtZXNwYWNlIjoiIiwibGFiZWwiOiJUb3AgUmVzdWx0cyIsImxhYmVsUGx1cmFsIjoiVG9wIFJlc3VsdHMiLCJyZXN1bHRzQ21wIjoiZm9yY2VTZWFyY2g6cHJlZGljdGVkUmVzdWx0cyJ9LCJjb250ZXh0Ijp7IkZJTFRFUlMiOnt9LCJzZWFyY2hTb3VyY2UiOiJBU1NJU1RBTlRfRElBTE9HIiwiZGlzYWJsZUludGVudFF1ZXJ5IjpmYWxzZSwiZGlzYWJsZVNwZWxsQ29ycmVjdGlvbiI6ZmFsc2UsInNlYXJjaERpYWxvZ1Nlc3Npb25JZCI6IjdkMWNhMmJhLWY1NzEtOTUyMy04ZGI1LTRmYWZiZmFlYTM3ZCIsImRlYnVnSW5mbyI6eyJhcHBOYW1lIjoiU3RhbmRhcmRfR1VTX2xlZnRfbmF2IiwiYXBwVHlwZSI6IlN0YW5kYXJkIiwiYXBwTmFtZXNwYWNlIjoiYyIsImxvY2F0aW9uIjoiZm9yY2U6b2JqZWN0SG9tZURlc2t0b3AiLCJzb2JqZWN0VHlwZSI6IkFETV9UaGlyZF9QYXJ0eV9Tb2Z0d2FyZV9fYyJ9fSwiZ3JvdXBJZCI6IkRFRkFVTFQifSwic3RhdGUiOnt9fQ%3D%3D), but not this latest version. 

I just submitted a request for version 7.22.5